tests/test_network.py

```diff
@@ -1600,3 +1600,16 @@ def test_connect_voltage_level_on_line():
     assert retrieved_splittedline2.loc['NHV1_NHV2_1_2', "voltage_level1_id"] == "N_VL"
     assert retrieved_splittedline2.loc['NHV1_NHV2_1_2', "voltage_level2_id"] == "VLHV2"
     assert retrieved_splittedline2.loc['NHV1_NHV2_1_2', "r"] == 0.75
+
+
+def test_add_load_bay():
```

Comment:
unfortunately I think sonar is right that we should add some tests for other types of elements as well ^^